stereo/depth_map_test.py

```diff
@@ -0,0 +1,54 @@
+import cv2
+import numpy
+from matplotlib import pyplot as plt
+
+
+def get_depth(img_1, img_2):
+
+
+    img_1_grey = cv2.cvtColor(img_1, cv2.COLOR_BGR2GRAY)
+    img_2_grey = cv2.cvtColor(img_2, cv2.COLOR_BGR2GRAY)
+
+    stereo = cv2.StereoBM_create(numDisparities=16, blockSize=15)
+    disparity = stereo.compute(img_1_grey, img_2_grey)
+
+    return disparity.astype("uint8")
```

Comment:
We need the `astype("uint8")` so we can display it with `imshow`. However if we're using depth for SLAM, we typically want the full precision computed from stereo matching. My suggestion would be to move the `astype` down into `imshow` so that `get_depth` does not lose any precision.